FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -633,8 +632,7 @@ def export_getComponentDocumentation( self, cType, system, module ):
     except Exception as _e:
       return S_ERROR( 'No documentation was found' )
 
-  types_storeHostInfo = []
-  def export_storeHostInfo( self ):
+  def __storeHostInfo( self ):
```

Comment:
Needs to be static method and then the self replaced by "SystemAdministratorHandler".
Also the gLogger.error below should get better information about who returned an error.